Tests/NimbleTests/Matchers/BeEmptyTest.swift

```diff
@@ -34,16 +34,12 @@ final class BeEmptyTest: XCTestCase, XCTestCaseProvider {
 
         expect("").to(beEmpty())
         expect("foo").toNot(beEmpty())
+
+        expect([] as NSSortOptions).to(beEmpty())
```

Comment:
It feels slightly weird to rely on `NSSortOptions` here, how do you feel about creating a `TestOptionSet` type for testing purposes?

Reply:
Makes sense :+1: